src/main.js

```diff
@@ -187,7 +187,7 @@ function createWindow(){
 
     // Create the browser window.
     mainWindow = new BrowserWindow({
-        width: 950 + (openDev ? 700 : 0), height: 625,
+        width: 950 + (openDev ? 700 : 0), height: 652,
```

Comment:
Why is the launcher size changed? It's been the same for years and I'm a bit worried changing the window size might cause issues it not fitting on all screens players might have.

Reply:
It's just a tiny bit, most people wouldn't probably even notice. This is to help the launch popup to not look a bit too much off-centered to the bottom, I mean if you're okay with that then this can be omitted, just a slight aesthetic change.